spec/ParseLiveQuery.spec.js

```diff
@@ -1269,4 +1269,25 @@ describe('ParseLiveQuery', function () {
     expect(object2.id).toBeDefined();
     expect(object3.id).toBeDefined();
   });
+
+  it('matchesKeyConstraints fails when subscribing a query with constraint notEqualTo null', async done => {
+    await reconfigureServer({
+      liveQuery: {
+        classNames: ['TestObject'],
+      },
+      startLiveQueryServer: true,
+      verbose: false,
+      silent: true,
+    });
+
+    const query = new Parse.Query(TestObject);
+    query.notEqualTo('foo', null);
+    await query.subscribe();
+
+    // Create a object and save it, this passes but server crashes afterwards
+    const object1 = new TestObject();
+    object1.set('foo', 'bar');
+    await object1.save();
+    done();
```

Comment:
Remove the `done();` and instead add an expect. The code subscribes to a query, so the expect should probably expect the event to be triggered.

Reply:
have added it. thanks for note.